src/main/java/org/dita/dost/ant/ExtensibleAntInvoker.java

```diff
@@ -199,6 +198,12 @@ public void execute() throws BuildException {
     private AbstractPipelineModule getPipelineModule(final ModuleElem m, final PipelineHashIO pipelineInput) throws DITAOTException {
         if (m instanceof XsltElem) {
             final XsltElem xm = (XsltElem) m;
+            if (xm.reloadstylesheet && xm.parallel) {
+                throw new BuildException("Both reloadstylesheet and parallel cannot be true");
```

Comment:
When in parallel mode, we can't reuse the XSLT executable, we have to get a new one from the XSLT template object for each thread.